src/components/TodoDeleteModalComponenet.vue

```diff
@@ -0,0 +1,113 @@
+<template>
```

Comment:
Can you create a separate general component for modal? You can add the title of the modal as header slot, and the body of the modal as a main slot.